frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -347,15 +347,24 @@ const PlatformWrapper = ({
 }`,
     };
 
-    let packageTypeHelpText = "";
+    let packageTypeHelpText;
```

Comment:
If not initializing to empty string, please define the type explicitly to avoid `any`:
`let packageTypeHelpText: string`

Reply:
Gotcha!